Lesson_06/homework_6.py

```diff
@@ -0,0 +1,37 @@
+# task 6.1
+proposed_string: str = input("Provide ur string here please: ")
+unique_counter: int = sum([1 for char in proposed_string if proposed_string.count(char) == 1])
+
+if unique_counter > 10:
+    print(True)
+else:
+    print(False)
+
+#rask 6.2
+
+is_correct_string: bool = False
+
+while not is_correct_string:
+    provided_string = str (input("Provide ur word pls")).lower()
```

Comment:
We do not need to do str() func for input - in case that input() func by default returns string type. I always says it on lessons. Please be more attentive -> for this - 2 points)